app/src/main/assets/drawable.xml

```diff
@@ -5654,4 +5654,5 @@
 	<item component="ComponentInfo{com.insideoutside.ihunteron/com.insideoutside.ihunter.iHunterActivity}" drawable="ihunter" />
 	<item component="ComponentInfo{au.com.redcat.chatime/au.com.redcat.chatime.MainActivity}" drawable="chatime" />
 	<item component="ComponentInfo{com.dhb.microrpg/com.unity3d.player.UnityPlayerActivity}" drawable="micro_rpg" />
+	<item component="ComponentInfo{com.formulaone.production/com.avs.f1.ui.splash.SplashActivity}" drawable="f1_tv" />
```

Comment:
This file is a little messed up, the line you add3ed and the ones above should usually not be here, we'll look into getting it fixed, but please be aware of it in the future.